cube/cube.c

```diff
@@ -1900,8 +1926,8 @@ static void demo_prepare_render_pass(struct demo *demo) {
         .pAttachments = attachments,
         .subpassCount = 1,
         .pSubpasses = &subpass,
-        .dependencyCount = 0,
-        .pDependencies = NULL,
+        .dependencyCount = 2,
+        .pDependencies = &attachmentDependencies,
```

Comment:
clang (correctly) complains about the & of the array  as opposed to the arrayname or the & of [0]